src/client/index.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 import { render } from 'react-dom'
-import { Router, match, browserHistory } from 'react-router'
+import { match, browserHistory } from 'react-router'
 import AsyncProps from 'async-props'
 import 'location-origin'
 import 'es6-promise/auto'
 import RouteWrapper from '../shared/route-wrapper'
 import config from 'tapestry.config.js'
 import mitt from 'mitt'
+import { AppContainer } from 'react-hot-loader'
+import Root from './root'
```

Comment:
From the docs `<AppContainer/> is a component that handles module reloading, as well as error handling. The root component of your app should be nested in AppContainer as a child. When in production, AppContainer is automatically disabled, and simply returns its children.`

I took it that it's probably removing itself from the bundle internally when the environment is `prod`, but I could be wrong. I'll have a look...

...I've looked, it includes this file in prod https://github.com/gaearon/react-hot-loader/blob/master/src/AppContainer.prod.js - A few extra bytes worth removing to be fair.